lib/import-service.js

```diff
@@ -113,13 +113,15 @@ ImportService.prototype._handlePage = function (error, json) {
   if (!json.features || !json.features.length || !json.features.length > 0) return
   self.count += json.features.length
   self.emit('progress', self.count)
-  esriToGeojson(self.fields, json, function (err, geojson) {
-    if (err) return self.emit('error', new Error('Error translating features to geojson'))
+  try {
+    var geojson = esriToGeojson(json, self.fields)
     var featureString = JSON.stringify(geojson.features).slice(1, -1)
```

Comment:
the catch needs to happen here. always make sure that try blocks are no larger than they absolutely need to be.